### PR TITLE
First review

### PR DESCRIPTION
**Pull request by recmo**

### CODE REVIEW

**Comment by rudolfix** [comment]
@Recmo please review my commits
I significantly simplified RoT and TimeSource contracts please review again

File: contracts/RoT.sol [hunk 1]
```diff
@@ -3,19 +3,19 @@ import "./Types.sol";
...
 contract RoT is Ownable {
     address public ESOPAddress;
-    address public addressOfCEO;
+    event ESOPAndCEOSet(address ESOPAddress, address ceoAddress);
 
-    // change esop contract
     function setESOP(address ESOP, address ceo) public onlyOwner {
-      addressOfCEO = ceo;
+      // owner sets ESOP and ceo only once then passes ownership to ceo
+      // initially owner is a developer/admin
```
Comment (recmo) [inline contracts/RoT.sol:10]: Why not do this in a constructor? `RoT(address ESOP, address owner)` where `owner` is the CEOs address.

**Comment by recmo** [comment]
I like that the contracts are more explicit, with inheriting `Ownable` and all the `safeMath`. Makes it easier to review!

**Comment by recmo** [comment]
I think next step would be to apply Solidty Style Guide. Solium is a great tool for linting Solidity files. And then we should review the ESOP contract.

File: contracts/RoT.sol (hunk 1 above)
Comment (rudolfix) [reply contracts/RoT.sol:10]: we do not have ESOP address when RoT is created, if this is what you mean. take a look at ESOP constructor. 
in the deployment process admin deploys both ESOP and RoT and passes ownership to CEO. I think I should keep CEO as separate role and do not transfer the ownership. Let me think.

**Pull request merged**